src/components/AnonymousChat/child/ChatReplyView.tsx

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable no-use-before-define */
+import React from 'react';
+import {StyleSheet, Text, View} from 'react-native';
+
+import dimen from '../../../utils/dimen';
+import {SIGNED} from '../../../hooks/core/constant';
+import {calculateTime} from '../../../utils/time';
+import {colors} from '../../../utils/colors';
+import {fonts, normalizeFontSize} from '../../../utils/fonts';
+
+interface ChatReplyViewProps {
+  type: 'SIGNED' | 'ANONYMOUS';
+  messageType: string;
+  replyData;
+}
+
+const ChatReplyView = ({type, messageType, replyData}: ChatReplyViewProps) => {
+  const isReply = messageType === 'reply';
+  const isReplyPrompt = messageType === 'reply_prompt';
+  if (!isReply && !isReplyPrompt) return null;
+  if (!replyData?.message) return null;
+
+  const userData = replyData?.user;
+
+  const containerStyle = [
+    styles.textContainer,
+    type === SIGNED ? styles.containerSigned : styles.containerAnon
+  ];
+
+  const textStyle = [styles.text, replyData?.message_type === 'deleted' && styles.deletedText];
```

Comment:
Why we not using constant variable for 'deleted' ?

Reply:
solved in new commit